README.md

```diff
@@ -29,6 +29,18 @@ curl https://cdn.rawgit.com/pimterry/notes/v0.1.1/notes > /usr/local/bin/notes &
 
 By default your notes live in ~/notes, but you can change that to anywhere you like by setting the `$NOTES_DIRECTORY` environmental variable.
 
+### Bash completion
+
+If you want bash autocompletion copy the completion script into the bash completion directory. The bash completion directory is `/usr/share/bash-completion/completions/` on my system, you can you can get the bash completion directory by running the following command:
```

Comment:
Can you replace 'on my system' with some quick details of your system? E.g. `on a typical [ubuntu/debian/arch] install`.